### ISSUE TITLE
Bug - Language switcher inefficient

### ISSUE THREAD

**Issue by npracht** [mautic/mautic#1811]
## Description
Try to change your account language to French, it is not taken into account (still in english).
I've tried to refresh, clear cache, logout/login. Nothing happens.

![capture d ecran 2016-05-30 a 14 42 48](https://cloud.githubusercontent.com/assets/14075239/15650763/4253a662-267b-11e6-95a3-73e83eb37d58.png)
![capture d ecran 2016-05-30 a 14 43 27](https://cloud.githubusercontent.com/assets/14075239/15650765/439e9888-267b-11e6-831d-6300c8d576a1.png)

**Comment by escopecz** [comment]
Hi Norman,

you asked at Slack chat some time ago how the translations get from Transifex to Mautic. Alan replied:

> We have a script that runs every 24 hours that will fetch all the translations that are greater than 80% (I think) translated and make them available as packages. We actually do not check if the translation has been marked as reviewed so trusting the community :smile: **Local translation files are not updated except during a Mautic upgrade or if you manually delete the local translation files forcing a refetch from the server.**

I highlighted the part which applies to this issue. Try to delete the /languages/fr directory in your Mautic and save the configuration again. Let me know if it worked.

**Comment by npracht** [comment]
Hi @escopecz ,

Here is a mautic.com account. There is no word in French. Like if changing wouldn't impact the language.

## What we've tried for local version
We've tried what you say for our local hosted version and languages are mixed up...
* For sms.smses we should have written `SMS`
* Text messages (english version) is still written

### In the code:
![pasted image at 2016_05_30 04_04 pm](https://cloud.githubusercontent.com/assets/14075239/15651566/baca6554-2680-11e6-98c2-df68730a22cc.png)

### In translation file:
![pasted image at 2016_05_30 04_05 pm](https://cloud.githubusercontent.com/assets/14075239/15651567/bb0d736c-2680-11e6-851b-24bbe02ba26d.png)

### Live on the app:
![capture d ecran 2016-05-30 a 16 09 26](https://cloud.githubusercontent.com/assets/14075239/15651605/ec43ca80-2680-11e6-8f60-8c2dac841552.png)

**Comment by escopecz** [comment]
I tried French on my local and the issue is with the SmsBundle/messages.ini file. I get this error:

```
500 Internal Server Error - Warning: syntax error, unexpected ')' in translations/fr/SmsBundle/messages.ini on line 29
```

The file around this line looks like this:
```
mautic.sms.form.action.sendsms.admin=Envoyer un SMS à l'utilisateur
; mautic.sms.form.action.sendsms.admin.descr="Send the selected sms to the selected user(s) upon form submission."
mautic.sms.form.action.sendsms.lead=Envoyer un SMS au prospect
mautic.sms.form.action.sendsms.lead.descr=Envoyer le SMS sélectionné au prospect lors de la soumission du formulaire.
mautic.sms.form.body=Corps
mautic.sms.form.confirmbatchdelete=Supprimer les SMS sélectionnés ?
mautic.sms.form.confirmdelete=Supprimer le SMS %name% ?
; mautic.sms.form.confirmsend="Queue, %name%, for sending?"
mautic.sms.form.internal.name=Nom interne
mautic.sms.form.list=Liste de prospects
```
Many translations are not within quotation marks. I thought it is not possible to make such issue in the Transifex UI.

@npracht could you please fix this at Transifex?

**Comment by mqueme** [comment]
I tried this too and I also get some errors in that same file: seems like these characters aren't accepted ()!

**Comment by npracht** [comment]
@escopecz so you mean, for all translated texts, i need to add `"` after and `"` before each group of word?

**Comment by escopecz** [comment]
Exactly. This is wrong:

`mautic.sms.form.body=Corps`

This is right:

`mautic.sms.form.body="Corps"`

**Comment by npracht** [comment]
@escopecz i've changed as it is in original language (english).
But, i sometimes have the english text with `' '`, sometimes with `" "` and sometimes with nothing.
Is that normal ?

![capture d ecran 2016-05-30 a 17 59 00](https://cloud.githubusercontent.com/assets/14075239/15653929/70812310-2690-11e6-84fc-9d38e2042f58.png)
![capture d ecran 2016-05-30 a 17 59 31](https://cloud.githubusercontent.com/assets/14075239/15653930/708352c0-2690-11e6-95f5-2c30390a5e0e.png)

**Comment by escopecz** [comment]
I think it cannot be fixed with the Transifex UI. It seems like someone had downloaded that file, translated it without the quotes and uploaded it back. It will have to be fixed the same way. I tried to do that, but when I uploaded the file back, it displayed some error for a second and then the whole SMSBundle file disappeared. I'm not sure what happened there. Could you take a look if you see it there? The fixed file is here:
[for_translation_mautic_smsbundle-messages_fr.ini.txt](https://github.com/mautic/mautic/files/290517/for_translation_mautic_smsbundle-messages_fr.ini.txt)

**Comment by npracht** [comment]
hi @escopecz 

For the french translation from transifex UI i can add either `"` or `'`. But i cannot change the english version. Here you'll have to manage it (with Don? I think he's the source of the SMS bundle right ?).

By my side, if you simply ask me to add `"` or `'` for all french words, i can proceed in 15 minutes i think it'll be corrected. Just tell me which symbol i should use.

**Comment by escopecz** [comment]
As I wrote before, it should be double quotes. Single quotes works as well, but they are used only by Don in one file so I'd rather use double quotes which are used everywhere. Could you please read again what I wrote and answer the question? It seems you didn't notice my last comment.

**Comment by npracht** [comment]
Corrected by #1817
